unfoldingWord/translationCore
# Feature - Issue #2216 - Makes FAB words clickable

#### This pull request addresses:

FAB labels not being clickable


#### How to test this pull request:

Go to the projects page, click the FAB button, click the labels. Clicking the labels should do the same as clicking the icons.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3028)
